src/main/java/ladder/domain/Name.java

```diff
@@ -0,0 +1,31 @@
+package ladder.domain;
+
+public class Name {
+    private final static int MAX_NAME_LENGTH = 5;
+    private final static String BLANK = " ";
+    private final String value;
```

Comment:
접근제한 순서도 [컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)이 있으니 참고해 주세요 
클래스 변수와 멤버 변수는 개행으로 가독성을 높여 보시면 좋을 것 같아요 
```suggestion
    private static final  int MAX_NAME_LENGTH = 5;
    private static final  String BLANK = " ";

    private final String value;
```